examples/nextjs-notification-settings/app/version-history-dialog.tsx

```diff
@@ -0,0 +1,108 @@
+import * as Dialog from "@radix-ui/react-dialog";
```

Comment:
Should we remove things like that from the example? Everything not related or needed for notification settings